src/main/java/tds/content/repositories/impl/S3ItemDataRepository.java

```diff
@@ -74,7 +75,12 @@ private String buildPath(final String itemDataPath) {
         final String dirName = file.getParentFile() == null
             ? ""
             : file.getParentFile().getName();
+        // If the directory name is not present, default to "items"
+        final String itemsOrStimuli = StringUtils.isEmpty(dirName)
```

Comment:
yes, its identical, including the mapping of the ITSDocument object. Items and Stimuli (referenced as "passages" in the code) both conform to the Itemrelease schema.